### PR TITLE
Add override for flask-mail

### PR DESCRIPTION
**Pull request by dhamaniasad**

### CODE REVIEW

File: caniusepython3/overrides.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "django-social-auth": "https://pypi.python.org/pypi/python-social-auth",
     "dnspython": "https://pypi.python.org/pypi/dnspython3",
     "extras": "https://pypi.python.org/pypi/extras",
+    "flask-mail": "https://github.com/mattupstate/flask-mail/blob/master/CHANGES#L24",
```
Comment (brettcannon) [inline caniusepython3/overrides.json]: Can you change this to `http://pythonhosted.org/Flask-Mail/changelog.html#version-0-9-0` so the link is stable?

Comment (dhamaniasad) [inline caniusepython3/overrides.json]: Done.

**Comment by brettcannon** [comment]
There are now merge conflicts. Could you fix them and ping the PR when its ready again?

**Comment by dhamaniasad** [comment]
The merge conflict has been fixed.

**Comment by brettcannon** [comment]
Thanks!

**Pull request merged**